Adyen/Service/Checkout/OrdersService.cs

```diff
@@ -34,18 +34,58 @@ public OrdersService(Client client) : base(client)
             _baseUrl = client.Config.CheckoutEndpoint + "/" + ClientConfig.CheckoutApiVersion;
         }
     
+        public CheckoutCreateOrderResponse Orders(CheckoutCreateOrderRequest checkoutCreateOrderRequest, RequestOptions requestOptions = default)
+        {
+            return OrdersAsync(checkoutCreateOrderRequest, requestOptions).GetAwaiter().GetResult();
+        }
+
+        public async Task<CheckoutCreateOrderResponse> OrdersAsync(CheckoutCreateOrderRequest checkoutCreateOrderRequest, RequestOptions requestOptions = default, CancellationToken cancellationToken = default)
+        {
+            var endpoint = _baseUrl + "/orders";
+            var resource = new ServiceResource(this, endpoint);
+            return await resource.RequestAsync<CheckoutCreateOrderResponse>(checkoutCreateOrderRequest.ToJson(), requestOptions, new HttpMethod("POST"), cancellationToken);
+        }
+
+        public CheckoutCancelOrderResponse CancelOrder(CheckoutCancelOrderRequest checkoutCancelOrderRequest, RequestOptions requestOptions = default)
+        {
+            return CancelOrderAsync(checkoutCancelOrderRequest, requestOptions).GetAwaiter().GetResult();
+        }
+
+        public async Task<CheckoutCancelOrderResponse> CancelOrderAsync(CheckoutCancelOrderRequest checkoutCancelOrderRequest, RequestOptions requestOptions = default, CancellationToken cancellationToken = default)
+        {
+            var endpoint = _baseUrl + "/orders/cancel";
+            var resource = new ServiceResource(this, endpoint);
+            return await resource.RequestAsync<CheckoutCancelOrderResponse>(checkoutCancelOrderRequest.ToJson(), requestOptions, new HttpMethod("POST"), cancellationToken);
+        }
+
+        public CheckoutBalanceCheckResponse GetBalanceOfGiftCard(CheckoutBalanceCheckRequest checkoutBalanceCheckRequest, RequestOptions requestOptions = default)
+        {
+            return GetBalanceOfGiftCardAsync(checkoutBalanceCheckRequest, requestOptions).GetAwaiter().GetResult();
+        }
+
+        public async Task<CheckoutBalanceCheckResponse> GetBalanceOfGiftCardAsync(CheckoutBalanceCheckRequest checkoutBalanceCheckRequest, RequestOptions requestOptions = default, CancellationToken cancellationToken = default)
+        {
+            var endpoint = _baseUrl + "/paymentMethods/balance";
+            var resource = new ServiceResource(this, endpoint);
+            return await resource.RequestAsync<CheckoutBalanceCheckResponse>(checkoutBalanceCheckRequest.ToJson(), requestOptions, new HttpMethod("POST"), cancellationToken);
+        }
+
+    }
+    
+    /// <summary>
+    /// Service Interface
+    /// </summary>
+    public interface IOrdersService
+    {
         /// <summary>
         /// Create an order
         /// </summary>
         /// <param name="idempotencyKey">A unique identifier for the message with a maximum of 64 characters (we recommend a UUID).</param>
```

Comment:
I don't see a idempotencyKey here (this is probably moved to the Client level (just a guess*), we can safely remove this comment 👍

Reply:
There are multiple functions/methods that have the same 'idempotencyKey'-comment ^ Not just this one*
Please remove it 👍

Reply:
yeah this is an annoying one tbh. Not really the scope of this PR but I'll see if I can remove it or if not create a ticket for it.